client/angular/components/thread_page/context_panel/context_panel.coffee

```diff
@@ -64,6 +63,11 @@ angular.module('loomioApp').directive 'contextPanel', ['$rootScope', ($rootScope
       icon: 'mdi-pin-off'
       canPerform: -> AbilityService.canUnpinThread($scope.discussion)
       perform:    -> ThreadService.unpin($scope.discussion)
+    ,
+      name: 'show_history',
+      icon: 'mdi-clock'
+      canPerform: -> $scope.discussion.edited()
```

Comment:
If we're not using edited() anywhere else I'd be happy to for this to simply be
```
$scope.discussion.versionsCount > 1
```
and to get rid of the `edited()` method